python/llm/example/transformers/transformers_int4/README.md

```diff
@@ -32,3 +32,27 @@ For better performance, it is recommended to set environment variables on Linux
 pip install bigdl-nano
 source bigdl-nano-init
 ```
+
```

Comment:
Maybe we still remove this section for now. As not only transformers examples need this, and the section order here is kind of strange.

Reply:
We may find another place for the conda installation, and refer to there later. (I feel like the official link is ok for now)

Reply:
> We may find another place for the conda installation, and refer to there later. (I feel like the official link is ok for now)

yeah, I agree. We can keep referring to official conda link in each example instead of add such a common part in https://github.com/intel-analytics/BigDL/blob/main/python/llm/example/transformers/transformers_int4/README.md.